commander/test/bootstrapping/commands/generator/export.spec.ts

```diff
@@ -34,7 +35,7 @@ describe('generator:export', () => {
 			maxHeightPrevoted: 1200,
 		},
 	];
-	const consoleWarnSpy = jest.spyOn(console, 'warn');
+	// const consoleWarnSpy = jest.spyOn(console, 'warn');
```

Comment:
Should this line be removed or uncommented?

Reply:
It's unnecessary now. Unused.